package.json

```diff
@@ -677,6 +676,7 @@
     "@kbn/search-examples-plugin": "link:examples/search_examples",
     "@kbn/search-index-documents": "link:packages/kbn-search-index-documents",
     "@kbn/search-response-warnings": "link:packages/kbn-search-response-warnings",
+    "@kbn/search_playground": "link:x-pack/plugins/search_playground",
```

Comment:
nitpick: Should we name it `@kbn/search-playground` to make sure it follows the same pattern?